core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt

```diff
@@ -105,7 +105,8 @@ fun LpEditFloatingActionButton(
     modifier: Modifier = Modifier,
     color: Color = MaterialTheme.colorScheme.primary,
     colors: Color = contentColorFor(backgroundColor = MaterialTheme.colorScheme.onPrimary),
-    onClick: () -> Unit,
+    elevation: FloatingActionButtonElevation,
```

Comment:
Please, add the new variable to the documentation

Reply:
Done!